channel.go

```diff
@@ -13,39 +13,54 @@ import (
 	"perun.network/go-perun/client"
 )
 
+// State wraps a go-perun/channel.State
+// ref https://pkg.go.dev/perun.network/go-perun@v0.2.2/channel?tab=doc#State
```

Comment:
You can actually omit the version tag, then it is more future proof
```suggestion
// ref https://pkg.go.dev/perun.network/go-perun/channel?tab=doc#State
```